src/internal/agent/http/admission/handler.go

```diff
@@ -112,12 +114,12 @@ func (h *Handler) Serve(hook operations.Hook) http.HandlerFunc {
 
 		jsonResponse, err := json.Marshal(admissionResponse)
 		if err != nil {
-			message.WarnErr(err, lang.AgentErrMarshalResponse)
+			l.Error("unable to marshal the response", "error", err)
 			http.Error(w, lang.AgentErrMarshalResponse, http.StatusInternalServerError)
 			return
 		}
 
-		message.Infof(lang.AgentInfoWebhookAllowed, r.URL.Path, review.Request.Operation, result.Allowed)
+		l.Info("webhook execution complete", "path", r.URL.Path, "operation", review.Request.Operation, "allowed", result.Allowed)
```

Comment:
Is this a good place to collect metrics on the Zarf agent, you think? Maybe count requests per url path?